astroid/_ast.py

```diff
@@ -1,21 +1,20 @@
 import ast
 import sys
+import types
 from collections import namedtuple
 from functools import partial
 from typing import Dict, Optional
 
 from astroid.const import PY38_PLUS, Context
 
-try:
-    import typed_ast.ast3 as _ast_py3
-except ImportError:
-    _ast_py3 = None
-
-
 if PY38_PLUS:
     # On Python 3.8, typed_ast was merged back into `ast`
-    _ast_py3 = ast
-
+    _ast_py3: Optional[types.ModuleType] = ast
+else:
+    try:
+        import typed_ast.ast3 as _ast_py3  # type: ignore[no-redef]
```

Comment:
```suggestion
        import typed_ast.ast3 as _ast_py3
```
With `sys.version_info`, you don't need the `type: ignore` anymore.